### PR TITLE
[RSDK-7790] Update examples

### PR DESCRIPTION
**Pull request by allisonschiang**
Updated examples to use "micrordk version 0.1.9-rc2"
removed anyhow and esp-idf-sys patch

### CODE REVIEW

File: src/wifi_rssi_sensor.rs [hunk 1]
```diff
@@ -48,13 +48,13 @@ impl Readings for WifiRSSISensor {
...
 impl SensorT<f64> for WifiRSSISensor {
-    fn get_readings(&self) -> anyhow::Result<TypedReadingsResult<f64>> {
+    fn get_readings(&self) -> Result<TypedReadingsResult<f64>, SensorError> {
         log::debug!("wifi-rssi sensor - get readings called");
         let mut ap_info = wifi_ap_record_t::default();
         unsafe {
             match esp_wifi_sta_get_ap_info(&mut ap_info as *mut wifi_ap_record_t) {
                 ESP_OK => {}
-                err => return Err(EspError::from(err).unwrap().into()),
+                err => return Err(SensorError::SensorCodeError(err)), //TODO: impl #from EspError variant
```
Comment (acmorrow) [inline src/wifi_rssi_sensor.rs:57]: Can we make a ticket for this so we don't forget, and update the TODO to reference it? Also, I'd put the TODO on the line prior:

```
// TODO(RSDK-WXYZ): Support #from EspError for SensorError
err => return Err(SensorError:...
```

**Review by acmorrow — APPROVED** [review]
LGTM with a ticket request

File: src/wifi_rssi_sensor.rs (hunk 1 above)
Comment (allisonschiang) [reply src/wifi_rssi_sensor.rs:58]: done!

**Review by acmorrow — APPROVED** [review]
LGTM

**Pull request merged**